taranis/__main__.py

```diff
@@ -184,29 +190,41 @@ def taranis_cli(verbose, log_file):
     default="Genus",
     help="Use genus-specific BLAST databases for Prokka schema genes annotation (needs --genus). Default is False.",
 )
+@click.option(
+    "--cpus",
+    required=False,
+    multiple=False,
+    type=int,
+    default=1,
+    help="Number of cpus used for execution",
+)
 def analyze_schema(
-    inputdir,
-    output,
-    remove_subset,
-    remove_duplicated,
-    remove_no_cds,
-    genus,
-    species,
-    usegenus,
+    inputdir: str,
+    output: str,
+    remove_subset: bool,
+    remove_duplicated: bool,
+    remove_no_cds: bool,
+    output_allele_annot: bool,
+    genus: str,
+    species: str,
+    usegenus: str,
+    cpus: int,
 ):
     schema_files = taranis.utils.get_files_in_folder(inputdir, "fasta")
 
     """
-    schema_analyze = {}
+    schema_analyze = []
```

Comment:
if this is testing code please add the info in a comment. Also tag "TODO. DELETE CODE" so we now this is meant to be deleted.

Reply:
Yes, it was for testing, Added tag for deleting,

Reply:
push the changes!